docs/src/content/docs/zh/principles/ssh.md

```
@@ -0,0 +1,304 @@
+---
+title: SSH 连接
+description: SSH 连接是如何建立和管理的
+---
+
+了解 Server Box 中的 SSH 连接机制。
+
+## 连接流程
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)